test/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ add_chimera_binding(
 
 set_target_properties(test_binding PROPERTIES PREFIX "")
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
Just out of curiosity, why is this flag necessary if the target compile options are set below? Is there some other target that is not getting the options?